### PR TITLE
fix: remove postman-code-generators constraint

### PR DESCRIPTION
**Pull request by jeffmaury**
### What does this PR do?

Remove postman-code-generators constraint

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#1827 

### How to test this PR?

`pnpm install`

### CODE REVIEW

**Review by axel7083 — APPROVED** [review]
LGTM

File: .github/workflows/build-next.yaml [hunk 1]
```diff
@@ -39,7 +39,9 @@ jobs:
           cache: 'pnpm'
 
       - name: Execute pnpm
-        run: pnpm install
+        run: | 
+          npm uninstall -g yarn
```
Comment (axel7083) [inline .github/workflows/build-next.yaml:43]: do we need to uninstall yarn ?

**Comment by benoitf** [comment]
how will it work within developers env ? Many folks may have noth yarn and pnpm installed
Will it cause an issue ?

**Comment by jeffmaury** [comment]
> how will it work within developers env ? Many folks may have noth yarn and pnpm installed Will it cause an issue ?

Yes will amend the upstream fix because I don't understand why they are running yarn when package.json specifies pnpm

**Pull request closed without merging**